src/main.rs

```diff
@@ -217,7 +134,7 @@ fn build(port: u16, outdir: &Path, sassbin: String) -> Result<()> {
     f.read_to_string(&mut content)
         .context("Couldn't read configuration `cuteconfig.toml`")?;
 
-    let config = toml::from_str::<Config>(&content).context("Couldn't parse configuration")?;
+    let config: Config = toml::from_str::<Config>(&content).context("Couldn't parse configuration")?;
```

Comment:
If you declare the type at the local, there's no need for this


```suggestion
    let config: Config = toml::from_str(&content).context("Couldn't parse configuration")?;
```

Reply:
fair